lib/openshift/flakes/container_spec.rb

```diff
@@ -22,6 +22,10 @@ def image
         return struct['image']
       end
 
+      def args
```

Comment:
The functions under the module are sorted.

Reply:
@liangxia that convention is not really followed in practice at all...we do encourage the sorting to be done alphabetically in the rules file.  I'll merge this to unblock @lihongyan1

Reply:
@pruan-rht  OK. Got it.